srcpkgs/tea/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'tea'
+pkgname=tea
+version=0.8.0
+revision=1
+wrksrc=$pkgname
+build_style=go
+go_import_path=code.gitea.io/tea
+depends="git"
+short_desc="Command line tool to interact with Gitea"
+maintainer="chili-b <dexter.gaonshatford@gmail.com>"
+license="MIT"
+homepage="https://gitea.io"
+changelog="https://gitea.com/gitea/tea/src/branch/master/CHANGELOG.md"
```

Comment:
```suggestion
changelog="https://gitea.com/gitea/tea/raw/branch/master/CHANGELOG.md"
```

Reply:
Done. I also changed it from the master branch to a tagged version.

Reply:
I would keep it at master, it's nicer because people can just click on the link in browser and they seem to merge all changelog changes back to master.

Reply:
> I would keep it at master

I changed it back.

Reply:
don't forget to link to the raw changelog

Reply:
Okay, I've fixed it for real this time.